go.mod

```diff
@@ -19,6 +19,7 @@ require (
 	github.com/grpc-ecosystem/grpc-gateway v1.16.0
 	github.com/linxGnu/grocksdb v1.8.0
 	github.com/pelletier/go-toml/v2 v2.0.6
+	github.com/pkg/errors v0.9.1
```

Comment:
it seems this package is unmaintained these days: https://github.com/pkg/errors

the cosmos-sdk's errors package provides a similar [`Wrap` method](https://pkg.go.dev/cosmossdk.io/errors#Wrap), albeit with some extra ABCI context.

Reply:
I'll consider using stdlib errors instead

Reply:
fixed